digitalocean/resource_digitalocean_monitor_alert.go

```diff
@@ -0,0 +1,291 @@
+package digitalocean
+
+import (
+	"context"
+	"log"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceDigitalMonitorAlert() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDigitalOceanMonitorAlertCreate,
+		ReadContext:   resourceDigitalOceanMonitorAlertRead,
+		UpdateContext: resourceDigitalOceanMonitorAlertUpdate,
+		DeleteContext: resourceDigitalOceanMonitorAlertDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"uuid": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Required: true,
```

Comment:
Running the example (good example btw) and it looks like "uuid: Cannot be both Required and Computed". I think we can keep it to just "Computed"

Reply:
Right, `uuid` should just be `Computed` and not `Required` as it is not user configurable and it always set from the API response.